logger.ts

```diff
@@ -0,0 +1,36 @@
+import fetch from 'cross-fetch';
+const _debounce = require('lodash/debounce');
+
+const ingestEndpoint = process.env.AXIOM_INGEST_ENDPOINT || process.env.NEXT_PUBLIC_AXIOM_INGEST_ENDPOINT;
+
+const debouncedSendLogs = _debounce(() => sendLogs(), 1000);
+
+let collectedLogs: any[] = [];
+
+function _log(level: string, message: string, args: any = {}) {
+  const l = { level, message, ...args };
```

Comment:
Please add a `_time` as you rightly noticed in the document.

Reply:
added it, but will think of a good way to fix the tests